package.json

```diff
@@ -1,6 +1,11 @@
 {
   "name": "dojo-codesandbox-template",
   "version": "1.0.0",
+  "scripts": {
+    "dev": "dojo build --mode dev --watch file --serve",
```

Comment:
I think this should be: ```dojo build --mode dev --watch --serve``` we don't have different watch modes anymore

Reply:
Fixed, but this is what a brand new package.json generated from `dojo create app` adds so we'll probably need to fix it there, too.